cpu/esp32/bootloader/Makefile

```diff
@@ -0,0 +1,295 @@
+  components/bootloader_support/src/bootloader_panic.c \
+  components/bootloader_support/src/bootloader_random.c \
+  components/bootloader_support/src/bootloader_random_$(CPU).c \
+  components/bootloader_support/src/bootloader_utility.c \
+  components/bootloader_support/src/$(CPU)/bootloader_$(CPU).c \
+  components/bootloader_support/src/$(CPU)/bootloader_sha.c \
+  components/bootloader_support/src/$(CPU)/bootloader_soc.c \
+  components/bootloader_support/src/esp_image_format.c \
+  components/bootloader_support/src/flash_encrypt.c \
+  components/bootloader_support/src/flash_partitions.c \
+  components/bootloader_support/src/flash_qio_mode.c \
+  components/bootloader_support/src/secure_boot.c \
+  components/efuse/$(CPU)/esp_efuse_fields.c \
+  components/efuse/$(CPU)/esp_efuse_table.c \
+  components/efuse/$(CPU)/esp_efuse_utility.c \
+  components/efuse/src/esp_efuse_api.c \
+  components/efuse/src/esp_efuse_fields.c \
+  components/efuse/src/esp_efuse_utility.c \
+  components/esp_common/src/esp_err_to_name.c \
+  components/esp_hw_support/compare_set.c \
+  components/esp_hw_support/cpu_util.c \
+  components/esp_hw_support/port/$(CPU)/chip_info.c \
+  components/esp_hw_support/port/$(CPU)/rtc_clk.c \
+  components/esp_hw_support/port/$(CPU)/rtc_clk_init.c \
+  components/esp_hw_support/port/$(CPU)/rtc_init.c \
+  components/esp_hw_support/port/$(CPU)/rtc_pm.c \
+  components/esp_hw_support/port/$(CPU)/rtc_sleep.c \
+  components/esp_hw_support/port/$(CPU)/rtc_time.c \
+  components/esp_hw_support/port/$(CPU)/rtc_wdt.c \
+  components/esp_rom/patches/esp_rom_crc.c \
+  components/esp_rom/patches/esp_rom_sys.c \
+  components/esp_rom/patches/esp_rom_tjpgd.c \
+  components/esp_rom/patches/esp_rom_uart.c \
+  components/esp_system/esp_err.c \
+  components/hal/cpu_hal.c \
+  components/hal/mpu_hal.c \
+  components/hal/wdt_hal_iram.c \
+  components/log/log_buffers.c \
+  components/log/log.c \
+  components/log/log_noos.c \
+  components/newlib/syscalls.c \
+  components/soc/$(CPU)/adc_periph.c \
+  components/soc/$(CPU)/dac_periph.c \
+  components/soc/$(CPU)/gpio_periph.c \
+  components/soc/$(CPU)/i2c_periph.c \
+  components/soc/$(CPU)/i2s_periph.c \
+  components/soc/$(CPU)/interrupts.c \
+  components/soc/$(CPU)/lcd_periph.c \
+  components/soc/$(CPU)/ledc_periph.c \
+  components/soc/$(CPU)/mcpwm_periph.c \
+  components/soc/$(CPU)/pcnt_periph.c \
+  components/soc/$(CPU)/rmt_periph.c \
+  components/soc/$(CPU)/rtc_io_periph.c \
+  components/soc/$(CPU)/sdio_slave_periph.c \
+  components/soc/$(CPU)/sdmmc_periph.c \
+  components/soc/$(CPU)/sigmadelta_periph.c \
+  components/soc/$(CPU)/spi_periph.c \
+  components/soc/$(CPU)/timer_periph.c \
+  components/soc/$(CPU)/touch_sensor_periph.c \
+  components/soc/$(CPU)/uart_periph.c \
+  components/soc/lldesc.c \
+  components/spi_flash/$(CPU)/spi_flash_rom_patch.c \
+  components/xtensa/eri.c \
+  components/xtensa/xt_trax.c \
+
+ifneq (,$(filter esp32,$(CPU)))
```

Comment:
```suggestion
ifeq (esp32,$(CPU))
```

Reply:
This is just temporary, in future makefile file version, we have something like:
```
ifneq (,$(filter esp32 esp32s3,$(CPU_FAM)))
  ESP_SDK_BOOTLOADER_SRCS += ...
```
To avoid unnecessary conflicts with each rebase, I would prefer to keep it that way for now.